AsyncDisplayKit/ASDisplayNode.mm

```diff
@@ -68,14 +69,20 @@ - (void)_staticInitialize;
 
 @implementation ASDisplayNode
 
-// these dynamic properties all defined in ASLayoutOptionsPrivate.m
-@dynamic spacingAfter, spacingBefore, flexGrow, flexShrink, flexBasis,
-         alignSelf, ascender, descender, sizeRange, layoutPosition, layoutableType;
+// Dynamic properties for ASLayoutables
+@dynamic layoutableType, size;
+
+// Dynamic properties for sizing
+@dynamic width, height, minWidth, maxWidth, minHeight, maxHeight;
+// Dynamic properties for stack spec
```

Comment:
nit : consistent line spacing here. either remove the blank line on 74 or add them between these comments.